README.md

```diff
@@ -77,7 +77,7 @@ See the [Table Of Contents](https://github.com/authorjapps/zerocode#table-of-con
 
 Maven and CI 🔨
 ====
-**Latest release: [1.2.x](https://search.maven.org/search?q=zerocode-rest-bdd)** 🏹
+**Latest release: [1.3.x](https://search.maven.org/search?q=zerocode-tdd)** 🏹
```

Comment:
General README comment, the last curly brace is cut a bit. Can u pls replace with a new one?

Reply:
Got it. Will fix 👍 .